concrete/src/Validator/Site/EnvironmentSpecificInterface.php

```diff
@@ -5,7 +5,7 @@
 interface EnvironmentSpecificInterface
 {
 
-    const PRODUCTION = 'prod';
+    const PRODUCTION = 'production';
```

Comment:
FWIW these are arbitrary and contained within this validator stack so this shouldn't matter.